elasticsearch-core/src/main/scala/com/sumologic/elasticsearch/restlastic/dsl/MappingDsl.scala

```diff
@@ -80,8 +80,9 @@ trait MappingDsl extends DslCommons {
 
   case class CompletionMapping(context: Map[String, CompletionContext]) extends FieldMapping {
     val _type = "type" -> "completion"
-    val _analzyer = "analyzer" -> "keyword"
-    val _sanalyzer = "search_analyzer" -> "keyword"
+    // simple analyzer does case insensitive autocomplete
+    val _analzyer = "analyzer" -> "simple"
```

Comment:
Updated CompletionMapping to use "keyword" as default and "simple" when caseSensitive is false